server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -805,15 +816,6 @@ export function configureWebSocketServices(
 									),
 								});
 
-								if (roomMap.size >= 1) {
```

Comment:
Was the removal of these lines intended?

Reply:
Yes! I replaced their functionality above with `baseLumberjackProperties`